### ISSUE TITLE
[Suggestion] Support Client side mod?

### ISSUE THREAD

**Issue by barraIhsan** [samolego/FabricTailor#22]
So we can /skin or press k to apply skin, but if the server doesn't have the mods, then your skin only visible to you and anyone else that have installed this mod too.

Is a good mod. Please add that

**Comment by samolego** [comment]
Yeah, it's on my todo list, however, I cannot promise anything, I guess it will be implemented with #12

**Comment by barraIhsan** [comment]
Oh also, if it support client side mod too, then is it possible to /cape? (work ONLY in singleplayer if it cant in multiplayer) just like #21
